.github/workflows/integ_test.yml

```diff
@@ -271,6 +270,501 @@ jobs:
+    needs: prepare-for-test
+    runs-on: macos-latest
+    environment: IntegrationTest
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+        with:
+          persist-credentials: false
+
+      - name: Make directory
+        run: mkdir -p ~/.aws-amplify/amplify-ios/testconfiguration/
+
+      - name: Copy integration test resouces
+        uses: ./.github/composite_actions/download_test_configuration
+        with:
+          resource_subfolder: datastore
+          aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          aws_region: ${{ secrets.AWS_REGION }}
+          aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG }}
+
+      - name: Run Integration test
+        uses: ./.github/composite_actions/run_xcodebuild_test
+        with:
+          project_path: ./AmplifyPlugins/DataStore/
+          workspace: DataStoreCategoryPlugin.xcworkspace
+          scheme: AWSDataStoreIntegrationHubTests
+  datastore-integration-local-test:
+    needs: prepare-for-test
+    runs-on: macos-latest
+    environment: IntegrationTest
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+        with:
+          persist-credentials: false
+
+      - name: Make directory
+        run: mkdir -p ~/.aws-amplify/amplify-ios/testconfiguration/
+
+      - name: Copy integration test resouces
+        uses: ./.github/composite_actions/download_test_configuration
+        with:
+          resource_subfolder: datastore
+          aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          aws_region: ${{ secrets.AWS_REGION }}
+          aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG }}
+
+      - name: Run Integration test
+        uses: ./.github/composite_actions/run_xcodebuild_test
+        with:
+          project_path: ./AmplifyPlugins/DataStore/
+          workspace: DataStoreCategoryPlugin.xcworkspace
+          scheme: AWSDataStoreIntegrationLocalTests
+
+  datastore-integration-V1-S1-test:
+    needs: prepare-for-test
+    runs-on: macos-latest
+    environment: IntegrationTest
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+        with:
+          persist-credentials: false
+
+      - name: Make directory
+        run: mkdir -p ~/.aws-amplify/amplify-ios/testconfiguration/
+
+      - name: Copy integration test resouces
+        uses: ./.github/composite_actions/download_test_configuration
+        with:
+          resource_subfolder: datastore
+          aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          aws_region: ${{ secrets.AWS_REGION }}
+          aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG }}
+
+      - name: Run Integration test
+        uses: ./.github/composite_actions/run_xcodebuild_test
+        with:
+          project_path: ./AmplifyPlugins/DataStore/
+          workspace: DataStoreCategoryPlugin.xcworkspace
+          scheme: AWSDataStoreIntegrationV1Scenario1Tests
+  datastore-integration-V1-S2-test:
```

Comment:
What does Vx Sx mean? Is there an easy way to understand what failed.

Reply:
Can we group them under a common parent?

Reply:
Vx is the version of the transformer and Sx is the scenario, this is just following the test class that it's enabling as close as possible, so changing the test class name should drive the change for these workflow names. One example is we could be more verbose if we want to to something like `TransformerV1ScenarioOneDirectionalHasOneTests`. (perhaps drop `AWSDataStoreIntegration` completely, and be explicit about the Transformer version and verbose about the scenario as closely as possible). I updated the nesting better as they all have one common parent which is the original `AWSDataStoreCategoryPluginIntegrationTests` that contains E2E tests